src/auditor/checks.py

```diff
@@ -462,21 +452,21 @@ def authoritative_check(self):
         self.results_dict.update(authoritative_data)
 
     def visibility_check(self):
-        '''
+        """
         Make sure item's default visibility is set to True
 
         Update results_dict with results for this item:
                 {'visibility_fix':''}
-        '''
+        """
 
-        fix_visibility = {'visibility_fix':'N'}
+        fix_visibility = {'visibility_fix': 'N'}
 
         for layer in self.item.layers:
-            
+
             #: Check if default vis is true; wrap in try/except for robustness
             try:
                 properties = json.loads(str(layer.manager.properties))
-            except:
+            except:  # pylint: disable=bare-except
                 properties = None
```

Comment:
**suggestion**: 
```
properties = None
try:
	properties = json.loads(str(layer.manager.properties))
except:
	pass
```

Another option that `pylint` might like better. Not sure that it is much better. Just an alternative thought.

Reply:
I think you'd still want an exception type. JSONDecodeError or AttributeError seem like the likely ones.

Reply:
Yeah, pylint isn't happy the other way (I still get the bare-except error). I really should go back and figure out what the error was that required the "robustness" in the first place; chances are, it gets taken care of with the retries one level up.

I still like the idea of setting `properties = None` first and then doing the check, though.

Reply:
Maybe the bare except is that you aren't catching a specific exception class...